doc/release-notes.md

```diff
@@ -267,6 +267,9 @@ Wallet: Pruning
 ---------------
 
 With 0.12 it is possible to use wallet functionality in pruned mode.
+This can reduce the disk usage from currently around 60 GB to
+less than 2 GB.
```

Comment:
I would prefer using "around 2 GB" to avoid an outdated document if utxo set, debug log, etc. grows.